tox.ini

```diff
@@ -2,83 +2,128 @@
 
 # Usage instructions:
 
-# `tox` will run all tests sequentially, `tox --parallel` will run all tests in parallel (much faster).
-# Run specific selection of tests with `tox -e pretest,<list-of-tests>,posttest` e.g., `tox -e pretest,test-api,test-launcher,posttest`
-# `--parallel` flag can be passed when running specific selections.
+# `tox` will run all "envlist" tests sequentially, `tox --parallel` same tests in parallel (much faster).
```

Comment:
@moe-ad once this is ready for use we should also document this in the `Contributing` section of the documentation.